packages/lodestar/test/e2e/network/network.test.ts

```diff
@@ -27,11 +27,12 @@ describe("[network] network", function () {
   this.timeout(5000)
   let netA: Libp2pNetwork, netB: Libp2pNetwork;
   const logger: ILogger = new WinstonLogger();
-  const metrics = new BeaconMetrics({enabled: true, timeout: 5000, pushGateway: false}, {logger});
+  logger.silent = true;
```

Comment:
@mpetrunic This line should silence the logger during E2E without relying just on the envar.